pkg/network/tracer/offsetguess.go

```diff
@@ -718,14 +718,20 @@ func newEventGenerator(ipv6 bool) (*eventGenerator, error) {
 
 	go acceptHandler(l)
 
+	// Spin up UDP server
+	udpAddr, udpDone, err := newUDPServer(addr)
```

Comment:
Is spinning up a server necessary? We don't actually need the server to respond.

Reply:
So you're suggesting we get rid of `net.Dial` altogether?

Reply:
Can we use `127.0.0.2` (not 127.0.0.1 to distinguish from the source) without spinning up a server?

Reply:
Oh, that might actually work. Let me try

Reply:
Yeah, that doesn't work :(
The `net.Dial` function in Golang actually does a `connect` syscall (even for UDP), and apparently that fails if you don't have an actual server on the other side. I'm not sure if there some ICMP magic going on to check that, but I guess an workaround would actually create a UDP socket ourselves and do something like a `sendto`, but I'm guessing this is more work than the code I have.

Reply:
Does that only fail if the address is loopback?